config/initializers/spotify.rb

```diff
@@ -0,0 +1,2 @@
+SPOTIFY_CLIENT_ID = ENV["SPOTIFY_CLIENT_ID"]
```

Comment:
If you use `#fetch` instead of `#[]`, you'll get notified more quickly about missing config values. Up to you whether you want the app to run without these set.